jerry-core/vm/vm.c

```diff
@@ -3395,15 +3427,26 @@ vm_execute (vm_frame_ctx_t *frame_ctx_p, /**< frame context */
 
   frame_ctx_p->stack_top_p = frame_ctx_p->registers_p + register_end;
 
-  if (arg_list_len > argument_end)
+#ifndef CONFIG_DISABLE_ES2015_FUNCTION_REST_PARAMETER
+  if (bytecode_header_p->status_flags & CBC_CODE_FLAGS_REST_PARAMETER)
```

Comment:
I would decrease argument_end in the parser. It would be useful for `length` property as well.